autometa.mf

```diff
@@ -0,0 +1,50 @@
+# Autometa Makeflow DAG
```

Comment:
Have you tested this to see if it works? Also, I don't know enough about makeflow to know if you made it so that calculating taxonomy is optional, or how that would be represented in a DAG.

Reply:
This was provided as an example. I have not tested this. Any user would need to populate the filepaths with their respective files

Reply:
This can be removed. Or perhaps moved to docs?